src/daemon/dlt-daemon.c

```diff
@@ -377,14 +397,7 @@ int option_file_parser(DltDaemonLocal *daemon_local)
     strncpy(daemon_local->flags.ctrlSockPath,
             DLT_DAEMON_DEFAULT_CTRL_SOCK_PATH,
             sizeof(daemon_local->flags.ctrlSockPath));
-#ifdef DLT_DAEMON_USE_UNIX_SOCKET_IPC
-    snprintf(daemon_local->flags.appSockPath, DLT_IPC_PATH_MAX, "%s/dlt", DLT_USER_IPC_PATH);
-
-    if (strlen(DLT_USER_IPC_PATH) > DLT_IPC_PATH_MAX)
```

Comment:
Could you please add this check again for both cases (using DLT_USER_IPC_PATH and optarg).

Reply:
Sorry, this commit was made quite long time ago, so I could be a bit out of scope. 
As I see we dont need here additional check, because we are using above s**n**printf() and str**n**cpy() which are limited by DLT_IPC_PATH_MAX. 
If this statement is correct, could you please elaborate why do you think those checks are needed here?

Reply:
Hello @vsloginov ,
sorry for the delay. I some how missed your message.
This check is to print a warning to stderr in case snprintf or strncpy is trimming the path due to size limites.

If I think about the if statement above, I think it is even not correct and has to be:
if(strlen(DLT_USER_IPC_PATH) +4 > DLT_IPC_PATH_MAX)
as we add the four characters "/dlt" to the string.

Does this make sense?